### DISCUSSION TITLE
Get default SWR configuration

### DISCUSSION THREAD

**Discussion by laymonage** [vercel/swr#1166] [Q&A] [answered]
Hi, is there a way to use the `defaultConfig` that's exported in [config.ts](https://github.com/vercel/swr/blob/master/src/config.ts)?

I'm trying to use a custom `onErrorRetry` handler if the `error` matches a specific criteria, but I also want to keep the default behavior (exponential backoff and other stuff) otherwise.

I want to avoid copying the code if that's possible. I tried to `import defaultConfig from 'swr/dist/config';` but it didn't work.

**Comment by koba04** [comment, marked as answer]
@laymonage You can get the `defaultConfig` through `SWRConfig.default`. https://github.com/vercel/swr/pull/1023

**Reply by laymonage** [reply]
Hey, that's exactly it! Thank you very much!
